### PR TITLE
Address #144 by clarifying that the metadata is opaque blob.

### PR DESCRIPTION
**Pull request by vaikas-google**
Also fix the 'bullets' section of examples to conform to what CF docs have for consistency.

### CODE REVIEW

**Comment by pmorie** [comment]
LGTM

**Comment by angarg12** [comment]
LGTM

File: spec.md [hunk 1]
```diff
@@ -197,13 +197,11 @@ A web-friendly display name is camel-cased with spaces and punctuation supported
...
             }
          ],
-        "bullets": [{
-            "content": "Shared fake server"
-          }, {
-            "content": "5 TB storage"
-          }, {
-            "content": "40 concurrent connections"
-        }]
+        "bullets": [
```
Comment (duglin) [inline spec.md]: why did you change this?  The old version was valid.

Comment (vaikas-google) [inline spec.md]: I tried to explain in the PR description, but, while this is a blob and yes, this is technically valid, it is:
 1. silly the way it's written
 2. does not comform to the existing Cloud Foundry convention, where bullets is an array of strings, not a map, so unless there's a strong reason for it to diverge, seemed like they should at least be consistent.
https://docs.cloudfoundry.org/services/catalog-metadata.html

Their examples are:
              "metadata":{
                  "bullets":[
                     "20 GB of messages",
                     "20 connections"
                  ],

Which makes more sense than duplicating 'content' over and over for no reason. It's confusing.

Comment (duglin) [inline spec.md]: ah ok - I missed the connection - thanks

**Comment by duglin** [comment]
LGTM

**Comment by avade** [comment]
Could we at least mention the existing CF community standards an FYI (https://docs.cloudfoundry.org/services/catalog-metadata.html) in the spec? 

Otherwise it seems fine.

**Comment by avade** [comment]
Could we at least mention the existing CF community standards an FYI (https://docs.cloudfoundry.org/services/catalog-metadata.html) in the spec? 

Otherwise it seems fine.

**Comment by vaikas-google** [comment]
I did mention that there are existing community standards, are you asking for a link to be included?

**Comment by pmorie** [comment]
I think a link to somewhere where the existing community standards are document would be very useful.

**Comment by vaikas-google** [comment]
Ok, added a link, ptal.

**Comment by avade** [comment]
lgtm

**Comment by angarg12** [comment]
It seems like pullapprove didn't pick up the LGTM, maybe we need to close and reopen the PR.

**Comment by duglin** [comment]
LGTM
agai

**Comment by duglin** [comment]
@pmorie need your LGTM again

**Comment by pmorie** [comment]
LGTM

**Comment by vaikas-google** [comment]
@avade Looks like you LGTMd it, but it didn't result in a pullapprove, is there something else that you need to do to convert that into pullapprove?
